<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPlayerMenu</class>
 <widget class="QDialog" name="CPlayerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headline">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Spielermenü&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="infoLayout">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Spielername:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="numberOfCountrys">
      <property name="text">
       <string>Anzahl besetzer Länder:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="percentageLabel">
      <property name="text">
       <string>in % der Landkarte</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="valueLayout">
    <item>
     <widget class="QLabel" name="nameInfo">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="numberOfCountrysInfo">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="percentageInfo">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="quitPB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
